RecoPixelVertexing/PixelLowPtUtilities/python/ClusterShapeHitFilterESProducer_cfi.py

```diff
@@ -11,6 +11,5 @@
 )
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase2_tracker.toModify(ClusterShapeHitFilterESProducer,
-    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase1Tk.par',
-    doPixelShapeCut = cms.bool(False)
+    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase2Tk.par',
```

Comment:
Let me remind that the most correct solution would be to move the whole calibration to GT (which would have saved some hairs with phase1 in pre12), but probably we are, again, after a "quicker" solution.